### PR TITLE
[BACKLOG-8624] - Make Localization Service available by default in products (Add to Feature)

### PR DESCRIPTION
**Pull request by marcoslarsen**
[BACKLOG-8624] - Make Localization Service available by default in products (Add to Feature). Bundle versions fix.

### CODE REVIEW

**Comment by marcoslarsen** [comment]
@gundisalwa @candentira @plagoa @francisco-guilherme @graimundo please review

**Comment by marcoslarsen** [comment]
@pentaho-nbaker @CodeOnCoffee please review

File: pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml [hunk 1]
```diff
@@ -255,5 +256,8 @@
         <!-- <feature version="${cxf.karaf.version}">cxf-features-clustering</feature> -->
         <bundle>mvn:commons-collections/commons-collections/${commons.collections.version}</bundle>
     </feature>
-
+    <feature name="pentaho-i18n" version="${project.version}">
+      <bundle>mvn:pentaho/pentaho-i18n-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
+      <bundle>mvn:pentaho/pentaho-i18n-webservice-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
```
Comment (pentaho-nbaker) [inline pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml]: We need to make the webservice bundle conditional based on the environment. It should only be installed when CXF is present. The following should do it

        <conditional>
      <condition>cxf</condition>
      <bundle>mvn:pentaho/pentaho-i18n-webservice-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
    </conditional>

**Comment by pentaho-nbaker** [comment]
Closing until webservice bundle is conditional

**Pull request closed without merging**